pcsd/capabilities.xml

```diff
@@ -2298,6 +2298,18 @@
         Resource agents implementing OCF 1.1 are supported.
       </description>
     </capability>
+    <capability id="resource-agents.simplified-query-api" in-pcs="0" in-pcsd="1">
+      <description>
+        Simplified API for listing and querying medatada of roesource/stonith
+        agents.
+
+        daemon urls:
+          /api/v1/resource-agent-get-agents-list/v1
+          /api/v1/resource-agent-get-agent-metadata/v1
+          /api/v1/stonith-agent-get-agents-list/v1
+          /api/v1/stonith-agent-get-agent-metadata/v1
+      </description>
+    </capability>
```

Comment:
OCF 1.1 capability is copied to stonith-agents. I think we want to copy this one as well.